### PR TITLE
feature/AP5-21

### PR DESCRIPTION
**Pull request by jeremy-roussel**

### CODE REVIEW

File: rigex-front-end/src/assets/account.svg [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+            </mask>
+            <use id="Mask" fill="#011627" fill-rule="nonzero" xlink:href="#path-1"></use>
+            <g id="Group" mask="url(#mask-2)">
+                <g transform="translate(-5.312500, -5.312500)" id="Color-/-Medium-Grey">
+                    <rect id="Rectangle" fill="#637F8E" x="0" y="0" width="31.875" height="31.875"></rect>
+                </g>
+            </g>
+        </g>
+    </g>
+</svg>
```
Comment (BarretteBanner) [inline rigex-front-end/src/assets/account.svg]: 👁️

File: rigex-front-end/src/assets/rigex-logo-small.svg [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            <path d="M46.1711923,19.4775736 C48.1291684,19.4775736 50.2458993,18.9219318 51.5159378,17.7841889 L49.796094,15.2441119 C49.0816973,15.8791311 47.8116588,16.3289364 46.6739159,16.3289364 C44.7688581,16.3289364 43.6575744,15.2441119 43.4194422,14.0534508 L52.521385,14.0534508 L52.521385,13.1802993 C52.521385,8.84100096 49.7696348,6.06279168 45.9859784,6.06279168 C41.9906488,6.06279168 39.1859804,9.05267405 39.1859804,12.7569531 C39.1859804,16.9110374 42.2287811,19.4775736 46.1711923,19.4775736 Z" id="Path" fill="#00263E" fill-rule="nonzero"></path>
+            <path d="M48.631892,11.4075372 L43.3665239,11.4075372 C43.5252787,10.4814674 44.2132163,9.21142886 45.9859784,9.21142886 C47.864577,9.21142886 48.5260554,10.5079265 48.631892,11.4075372 Z" id="Path" fill="#FFFFFF" fill-rule="nonzero"></path>
+            <polygon id="Path" fill="#00263E" fill-rule="nonzero" points="56.622551 19.160064 59.1097098 15.2970301 61.5704095 19.160064 66.0420035 19.160064 61.676246 12.5981983 65.750953 6.38030131 61.279359 6.38030131 59.1097098 9.84644813 56.9136015 6.38030131 52.4420076 6.38030131 56.5431736 12.5981983 52.1774162 19.160064"></polygon>
+        </g>
+    </g>
+</svg>
```
Comment (BarretteBanner) [inline rigex-front-end/src/assets/rigex-logo-small.svg]: 👁️

File: rigex-front-end/src/assets/search.svg [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+            </mask>
+            <use id="Mask" fill="#011627" fill-rule="nonzero" xlink:href="#path-1"></use>
+            <g id="Group" mask="url(#mask-2)">
+                <g transform="translate(-3.000000, -3.000000)" id="Color-/-Medium-Grey">
+                    <rect id="Rectangle" fill="#637F8E" x="0" y="0" width="24" height="24"></rect>
+                </g>
+            </g>
+        </g>
+    </g>
+</svg>
```
Comment (BarretteBanner) [inline rigex-front-end/src/assets/search.svg]: 👁️

File: rigex-front-end/src/components/NavBar.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+import { makeStyles } from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import IconButton from '@material-ui/core/IconButton';
+import Button from '@material-ui/core/Button';
+import logo from "../assets/rigex-logo-small.svg";
+import search from "../assets/search.svg";
+import account from "../assets/account.svg";
+
+const useStyles = makeStyles(() => ({
```
Comment (BarretteBanner) [inline rigex-front-end/src/components/NavBar.js]: Let's move the styles into a file of it's own

File: rigex-front-end/src/components/NavBar.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
+import React from 'react';
```
Comment (BarretteBanner) [inline rigex-front-end/src/components/NavBar.js]: please follow import convention

File: rigex-front-end/src/components/NavBar.js [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+}));
+
+export default function NavBar() {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.navBar}>
+      <AppBar position="sticky" color="inherit">
+        <Toolbar className={classes.toolbar}>
+          <Button href="/landing" color="inherit" >
```
Comment (BarretteBanner) [inline rigex-front-end/src/components/NavBar.js]: Let's use the object used to get paths in the routes

File: rigex-front-end/src/components/NavBar.js [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+import IconButton from '@material-ui/core/IconButton';
+import Button from '@material-ui/core/Button';
+import logo from "../assets/rigex-logo-small.svg";
+import search from "../assets/search.svg";
+import account from "../assets/account.svg";
+
+const useStyles = makeStyles(() => ({
+  navBar: {
+    flexGrow: 1,
+    background: 'white',
```
Comment (BarretteBanner) [inline rigex-front-end/src/components/NavBar.js]: Let's use css variables for all colors. The color variables should be stored in a file and imported into where you use them in your styles. You may have to do some googling for this

**Review by BarretteBanner — APPROVED** [review]
This is really good Jeremy! 💯

**Review by mmevans — APPROVED** [review]

**Review by BarretteBanner — APPROVED** [review]

**Pull request merged**